CONTRIBUTING.md

```diff
@@ -59,6 +69,14 @@ Milestones are used to manage sprints, which are two weeks long. Sprint mileston
 - **13:** Issues that are more complicated and need some workflow or design planning. These issues usually need additional unit tests written.
 - **40:** If an issue is this complicated it should be converted into an epic.
 
+### Epics
+
+Epics are specified by the `epic` label. Epics are changes that require a lot of work and wouldn't fit into a single milestone. An epic should be a single concept, and have child issues for individual tasks created and listed in the epic's issue body.
+
+### Labels
+
+GitHub labels are used for organizing issues and providing context. You should read through the [label descriptions](https://github.com/Esri/calcite-components/labels) to familiarize yourself with what they signify.
```

Comment:
A suggestion for the second sentence to be less on the reader:

You [can familiarize yourself with]  ~~should read through~~ the label descriptions ~~to familiarize yourself with what they~~ [to understand what they] signify.

Reply:
Should we mention these sentences earlier on in the doc since labels are mentioned prior to this section?